proxies/nodejs/app.ts

```diff
@@ -3,10 +3,10 @@ import Router from 'koa-router'
 import bodyParser from 'koa-bodyparser'
 import { handleUser } from './handlers/user'
 import { handleClient } from './handlers/client'
-import { validateLocationRequest } from './handlers/location'
+import { handleLocation, validateLocationReqMiddleware } from './handlers/location'
 import { Data } from './entityTypes'
 
-const data: Data = {
+export const dataStore: Data = {
```

Comment:
could you rename the type to `DataStore` as well?

Reply:
that seems reasonable